### ISSUE TITLE
How do I use this alt_cuda_corr during train 

### ISSUE THREAD

**Issue by aiforworlds** [princeton-vl/RAFT#92]
Hi! Thanks for your great work on RAFT ，
     I was very interested in your alt_cuda_corr and did some experiments, But the results of the experiment were not correct, So there are a few very confused place want to consult, In my experiment, the alt_cuda_corr was first compiled, and the alt_cuda_corr mode was selected（line104-line107 in RAFT/core/raft.py） for training after successful compilation. There were no mistakes in the training, but when i running demo.py  with the --alternate_corr the result is not correct, The whole result picture is blurry, Can you give me some introductions and suggestions about the alt_cuda_corr use,  Thanks a lot.

**Comment by zachteed** [comment]
Hi, are you training with the '--alternate_corr` flag or just using it during inference. 

Currently, only the forward pass of the alt_cuda_corr layer is implemented so it can only be used for inference right now.

**Comment by aiforworlds** [comment]
yes I have trained with  the '--alternate_corr` flag, but  I cannt understand if you haven't train with alt_cuda_corr ,but you can used alt_cuda_corr for inference without train with alternate_corr layer, is it right? Thanks a lot.

**Comment by az-ja** [comment]
@[aiforworlds](https://github.com/aiforworlds) You can use it only for inference, since the the computed correlation using the original and the alternative code is identical. You should train with the alternative option, since it does not have the backward pass implemented and the results will be much worse, as a result.
(@zachteed  sorry for jumping in :) )
